### ISSUE TITLE
Documentation: Chrome extensions tutorial missing downloads

### ISSUE THREAD

**Issue by conan** [GoogleChrome/developer.chrome.com#138]
**Describe the bug**
Downloads that support the tutorial are missing, for example

the popup.html file is linked to this URL which does not exist: https://developer.chrome.com/docs/extensions/mv3/getstarted/examples/tutorials/get_started/popup.html

the images folder containing icons is linked to this URL which does not exist: https://developer.chrome.com/docs/extensions/mv3/getstarted/examples/tutorials/get_started/images.zip

**To Reproduce**
1. Go to https://developer.chrome.com/docs/extensions/mv3/getstarted/
2. Click on 'Download the images folder here' link - the "here" is the link
4. See 404 error

**Expected behavior**
Files should be available for download

**Screenshots**

![image](https://user-images.githubusercontent.com/3037019/103171641-ca5d7d80-4845-11eb-8d6e-b27767eb8542.png)

![image](https://user-images.githubusercontent.com/3037019/103171635-bd408e80-4845-11eb-96a4-348e4d0db388.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Chrome
 - Version 87.0.4280.88 (Official Build) (64-bit)

**Additional context**
Looks like this is related to the move to manifest v3 for extensions - the example code no longer works as the manifest version has been increased but the manifest entries have not been updated to stay compatible (e.g. "scripts" being replaced with "service_worker")

**Comment by leesw1347** [comment]
i'im getting the same error

**Comment by drwho85** [comment]
https://github.com/GoogleChrome/chrome-extensions-samples/tree/master/tutorials/get_started

**Comment by leesw1347** [comment]
> https://github.com/GoogleChrome/chrome-extensions-samples/tree/master/tutorials/get_started

so nice

**Comment by riccardomessineo** [comment]
> https://github.com/GoogleChrome/chrome-extensions-samples/tree/master/tutorials/get_started

Actually it's the old MV2... 
They actually deployed an half-done release... pretty odd! 
Are the MV3 out? Are they not?

**Comment by drwho85** [comment]
I assume they're just not done transitioning over to v3. You can replace the "mv3" in the url's to "mv2" and get the old stuff. That should work for a while. https://developer.chrome.com/docs/extensions/mv2/getstarted/.

**Comment by lukewis** [comment]
I'm new to extension development, and the current tutorial (being split between V2 and V3 with many of the links broken) is nearly unusable.  Hopefully once the transition is complete, there will be a regression over the tutorial as a whole.

**Comment by marco-meini** [comment]
In addition to missing downloads some scripts in the documentation contain errors, both about basic programming as well as version of manifest.

this is an axample:
https://developer.chrome.com/docs/extensions/mv3/getstarted/#logic

in the samples repository there is the correct code:
https://github.com/GoogleChrome/chrome-extensions-samples/blob/master/tutorials/get_started_complete/popup.js

**Comment by dotproto** [comment]
Closing this issue as I believe this was fixed in PR #317 and further refined in subsequent changes. If you encounter any issues with the [MV3 getting started guide](https://github.com/GoogleChrome/developer.chrome.com/blame/main/site/en/docs/extensions/mv3/getstarted/index.md) please file a new issue.
